backend/src/middleware/sentryMiddleware.js

```diff
@@ -14,13 +14,16 @@ if (sentryConfigs.SENTRY_DSN_BACKEND) {
     },
     withScope: (scope, error, context) => {
       scope.setUser({
-        id: context.user.id,
+        id: context.user && context.user.id,
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-23T11:18:59Z" title="Friday, August 23rd 2019, 1:18:59 pm +02:00">Aug 23, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -14,7 +14,7 @@ if (sentryConfigs.SENTRY_DSN_BACKEND) {
     },
     withScope: (scope, error, context) => {
       scope.setUser({
-        id: context.user.id,
+        id: context.user && context.user.id,
```


That's the actual fix.